includes/templates/responsive_classic/css/stylesheet_css_buttons.css

```diff
@@ -1,22 +1,19 @@
-#advSearchDefault input.cssButton.submit_button.button.button_search, #advSearchDefault .button_search:hover{font-size:1.3em;}
+.buttonRow a { text-decoration: none; }
+.cssButton {display:inline-block;font-size:1.3em;margin:0;padding:8px 20px;text-decoration:none;}
+input.submit_button {border:none;font-size: 1.2em;display: inline-block;margin:0;padding: 12px 30px 30px 30px;}
+input.submit_button:hover {border:none;font-size: 1.2em;display: inline-block;margin:0;padding: 12px 30px 30px 30px;}
+input.cssButtonHover {border:none;cursor: pointer;border:none;font-size: 1.2em;display: inline-block;margin:0;padding: 12px 30px 30px 30px;}
+span.cssButton.normal_button.button_more_reviews, .button_more_reviews:hover, span.cssButton.normal_button.button.button_read_reviews, .button_read_reviews:hover{display:block;text-align:center;}
+span.cssButton.normal_button.button_write_review, .button_write_review, span.cssButton.normal_button.button_in_cart{display:block;text-align:center;padding:12px 20px;}
+#indexBody span.cssButton.normal_button.button_in_cart, #indexBody .button_in_cart:hover{display:inline-block;}
+span.cssButton.normal_button.bbutton_goto_prod_details{background:#000;}
+.button_goto_prod_details:hover{background:#05a5cb;}
+input.cssButton.submit_button.button_search, .button_search:hover{padding:8px 20px !important;font-size:1.0em;line-height:18px;}
+#advSearchDefault input.cssButton.submit_button.button_search, #advSearchDefault .button_search:hover{font-size:1.3em;}
 .button-left{margin-right:10px;}
 .button-right{margin-left:10px;}
-span.cssButton.normal_button.button.button_checkout, .button_checkout:hover{display:block;padding:15px 30px;display:block;}
-span.cssButton.normal_button.button.button_continue_shopping, .button_continue_shopping:hover, span.cssButton.normal_button.button.button_shipping_estimator, .button_shipping_estimator:hover, span.cssButton.normal_button.button.button_back, .button_back:hover{}
-#reviewsInfoDefault span.cssButton.normal_button.button.button_in_cart, #reviewsInfoDefault .button_in_cart:hover{display:block;text-align:center;padding:12px 20px !important;}
-#reviewsInfoDefault .button_in_cart:hover, #reviewsDefault .button_in_cart:hover{display:block !important;}
+span.cssButton.normal_button.button_checkout, .button_checkout:hover{display:block;padding:15px 30px;display:block;}
+#reviewsInfoDefault span.cssButton.normal_button.button_in_cart, #reviewsInfoDefault .button_in_cart:hover{display:block;text-align:center;padding:12px 20px;}
+#reviewsInfoDefault .button_in_cart:hover, #reviewsDefault .button_in_cart:hover{display:block;}
 #reviewsDefault .button_in_cart:hover{text-align:center;padding:12px 20px;}
-span.cssButton.normal_button.button.button_logoff, span.cssButton.normal_button.button.small_edit{background:#666;}
+span.cssButton.normal_button.button_logoff, span.cssButton.normal_button.button.small_edit{background:#666;}
```

Comment:
And `.button` here in the end of this line? 
`, span.cssButton.normal_button.button.small_edit{...`

Reply:
crap. Do I pull the pr? Back to not knowing what to do or how to do it.

Reply:
You can make another commit with these changes, in the same branch, and push that to this same branch in Github, and it will add that commit to this PR.

- looks like you called this branch `CSS-button-cleanup`
- make sure you're in the `CSS-button-cleanup` branch on your PC
- make the fixes to the relevant files
- "stage" them (add the changes to a new commit), and commit them. 
- push the local `CSS-button-cleanup` branch to your github `CSS-button-cleanup` branch